### ISSUE TITLE
If a theme is applied, use the theme's token in the `Tag` component

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1098]
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.0

### Description

Currently, styling values in components ignore theme tokens. If a theme is applied, you should use the theme's token.

Yamada UI's style system provides fallback functions. If the value of the token in the first argument does not exist, it returns the second argument, the fallback value.

```diff
const css: CSSUIObject = {
  maxW: "100%",
  display: "inline-flex",
  alignItems: "center",
-  gap: "0.25rem",
+  gap: "fallback(1, 0.25rem)",
  verticalAlign: "top",
  ...styles.container,
}
```

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

**Comment by GensIto** [comment]
@hirotomoyamada 
Can I work on this issue?

**Comment by hirotomoyamada** [comment]
@GensIt Thanks!

**Comment by hirotomoyamada** [comment]
Fixed #1106
